<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExDateTime</class>
 <widget class="QWidget" name="ExDateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日期时间（QTime、QDate、QDateTime）</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>日期时间</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTimeEdit" name="timeEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>日期</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="editDate"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labCurrDataTime">
          <property name="text">
           <string>当前时间：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="editTime"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>时间</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="editDateTime"/>
        </item>
        <item row="2" column="1">
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="QDateTimeEdit" name="dateTimeEdit"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="btnGetDateTime">
          <property name="text">
           <string>获取当前时间日期</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>定时器（QTimer）</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="1">
         <widget class="QLCDNumber" name="lcdmm"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btnPeriod">
          <property name="text">
           <string>设置周期</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labGo">
          <property name="text">
           <string>时间已流逝：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnStatrt">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>定时周期</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnStop">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLCDNumber" name="lcdHH"/>
        </item>
        <item row="2" column="2">
         <widget class="QLCDNumber" name="lcdSS"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox">
          <property name="maximum">
           <number>10000000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
